Text/Hamlet.hs

```diff
@@ -334,6 +334,9 @@ ihamletRules = do
           urender $ \ur' -> mrender $ \mr -> return (e `AppE` mr `AppE` ur')
     em _ _ = error "bad Env"
 
+ixhamlet :: QuasiQuoter
```

Comment:
Can you add a short Haddock comment explaining what this quasiquoter does, and ideally a `@since` line stating the version it's added?